### PR TITLE
adds some more params to docker startup command

### PR DESCRIPTION
**Pull request by gregjan**
moves docker compose stack configurations to a subdirectory
adds profiles to webapp for inclusion of JMS and WebAC (over BASIC)
adds a "full stack" docker configuration w/JMS and WebAC

### CODE REVIEW

File: webapp/pom.xml [hunk 1]
```diff
@@ -407,5 +400,40 @@
...
+    <profile>
+      <id>jms</id>
+      <!-- Adds JMS-based EventService implementation-->
+      <dependencies>
+        <dependency>
+          <groupId>org.trellisldp</groupId>
+          <artifactId>trellis-jms</artifactId>
+          <version>${trellis.version}</version>
+        </dependency>
+        <dependency>
```
Comment (ajs6f) [inline webapp/pom.xml]: Does `trellis-jms` not have a dependency on `trellis-event-serialization`? @acoburn should it have?

**Comment by gregjan** [comment]
I wasn't aware that we changed anything with ENV vars yet. Is that in the
master branch now?  I suppose JMS should depend on event-serialization in a
transitive way.. (Better than my fix I think.)

**Comment by ajs6f** [comment]
Yeah, the ENV var stuff was in https://github.com/trellis-ldp/trellis-cassandra/commit/c8e2652b0bfbd24a6a269eba00237ec74ae96261.

**Comment by gregjan** [comment]
Many thanks. I will try a revised startup script.

File: webapp/pom.xml (hunk 1 above)
Comment (acoburn) [reply webapp/pom.xml]: No, there's no direct dependency. The service loader just tries to load some implementation of `ActivityStreamService` (`trellis-event-serialization` is one such implementation). This makes it possible to swap that out if you want some different type of serialization.

**Comment by ajs6f** [comment]
@gregjan Should we be moving to merge this, or are you still tinkering with that revised script? No hurry, just don't want to let it get stale.

**Comment by gregjan** [comment]
I am in favor of merging, just so that it keeps the current traceability in
tests, by saving the commit ID. I'll follow up with testing of direct to
Trellis ENV config, which will eliminate much of the startup script
complexity.

**Comment by ajs6f** [comment]
Cool, and it is ready to merge now?

**Comment by gregjan** [comment]
I added another commit and it is ready to merge.

**Pull request merged**